css/properties/-webkit-text-combine.json

```
@@ -5,7 +5,7 @@
         "__compat": {
           "support": {
             "chrome": {
-              "version_added": "≤80"
+              "version_added": "≤15"
```

See: https://github.com/caugner/css-properties/commit/127ade0b57e059a9580f990aa2acbf24a6db059d

```suggestion
              "version_added": "9"
```